client/src/views/admin/components/WeekMonitor.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <div>
+    <h2 class="title">
+      <strong>
+        {{ nameCenter }}
+      </strong>
+    </h2>
+    <carousel
+      id="carousel"
+      class="carousel"
+      :perPage="5"
+      :navigationEnabled="true"
+      :paginationEnabled="false"
+      :scrollPerPage="false"
+      :perPageCustom="[[768, 5], [1024, 5]]"
+      :navigateTo="[currentWeekNumber, false]"
+    >
+      <slide
+        class="slide"
+        v-ripple
+        v-for="(week, index) in formatArrayByWeek()"
+        :key="week.numWeek"
+      >
+        <v-card :id="`week-${nameCenter}-${index}`" class="main-card" @click="goToGestionPlannings">
+          <div class="week-card">
+            <v-card-text>S{{ index }}</v-card-text>
+          </div>
+          <v-divider/>
+          <div class="stats-card">
+            <v-card-text class="stats-card-text">
+                {{ week.freePlaces || 0 }} / {{ week.totalPlaces || 0 }}
+            </v-card-text>
+          </div>
+        </v-card>
+      </slide>
+    </carousel>
+  </div>
+</template>
+
+<script>
+import { DateTime } from 'luxon'
+
+export default {
+  props: {
+    nameCenter: {
+      type: String,
+      default: '',
+    },
+    weeks: {
+      type: Object,
+      default () {
+        return {}
+      },
+    },
+  },
+
+  computed: {
+    currentWeekNumber () {
+      return DateTime.local().setLocale('fr').weekNumber
+    },
+  },
+
+  methods: {
+    goToGestionPlannings () {
+      this.$router.push({ name: 'gestion-plannings' })
+    },
+
+    formatArrayByWeek () {
+      const allWeeks = Array(53).fill(false)
+      Object.keys(this.weeks).map(item => {
+        allWeeks[item] = {
+          days: [ ...this.weeks[item] ],
+          numWeek: item,
+          totalPlaces: this.weeks[item].length,
+          freePlaces: this.weeks[item].length - this.weeks[item].filter(elmt => elmt.candidat).length,
```

Comment:
`this.weeks[item].filter(elmt => !elmt.candidat).length` ne revient pas au même résultat ?